### ISSUE TITLE
ETag MD5 match issues

### ISSUE THREAD

**Issue by mzealey** [nyaruka/rp-archiver#87]
Most of our S3 uploads are having issues with etag vs md5 calcs being incorrect, for example:

```
time="2023-09-15T00:07:08Z" level=error msg="error deleting archive" archive_id=8315 count=1064179 error="archive md5: c881dde72956e6093c0c6ba422c9b9b0 and s3 etag: 7b40ab3cd948ed0d657b49c16f45f2eb do not match" org_id=25 period=D start="2023-06-08 00:00:00 +0000 UTC" type=message
```

It seems from https://docs.aws.amazon.com/AmazonS3/latest/API/API_Object.html that ETag is calculated quite differently depending on upload size, S3 encryption setting etc. I suspect having our buckets by default encrypted is causing this issue.

https://github.com/nyaruka/rp-archiver/issues/45 seems a related issue.

In short I'm not sure that MD5 ETag checks are the correct way to go here. Perhaps it's worth adding a flag to make this check optional? Because actually for us, the danger of having too much stuff in the DB and not archived (causing space usage, slowdowns etc) is likely greater than the cost of loosing old history.

**Comment by rowanseymour** [comment]
Addressed in https://github.com/nyaruka/rp-archiver/pull/90
